packages/clerk-js/src/ui/components/UserButton/useMultisessionActions.tsx

```diff
@@ -35,9 +35,11 @@ export const useMultisessionActions = (opts: UseMultisessionActionsParams) => {
 
   const handleManageAccountClicked = () => {
     if (opts.userProfileMode === 'navigation') {
-      return navigate(opts.userProfileUrl).finally(async () => {
-        await sleep(300);
-        opts.actionCompleteCallback?.();
+      return navigate(opts.userProfileUrl || '').finally(() => {
+        void (async () => {
+          await sleep(300);
+          opts.actionCompleteCallback?.();
```

Comment:
🙃  should we also await `opts.actionCompleteCallback?.();`. Note for separate commit or PR.
Also do we need the `void(` since we are not using return in `finally`.